backend/src/api/v1/yourApiName/middleware/rateLimiter.ts

```diff
@@ -1,13 +0,0 @@
-import rateLimiter from "express-rate-limit";
-
-export const limiter = rateLimiter({
-	max: 3,
-	windowMs: 60000,
-	standardHeaders: false,
-	message: {
-		status: 429,
-		message:
-			"We cant take anymore requests at the moment - Please try again in a few minutes",
-		type: "error"
-	}
-});
```

Comment:
Lets keep this for now. We may need it as middleware for future endpoints so users can't spam our server